package.json

```diff
@@ -1,16 +1,16 @@
 {
   "license": "Unlicensed",
   "private": true,
-  "repository": {
-  },
+  "repository": {},
   "dependencies": {
     "babel-brunch": "~6.0.0",
+    "babel-preset-es2015": "^6.9.0",
```

Comment:
Would have thought that would be added into the package.json by default then. :o  Nice catch. I haven't used any JavaScript in the project so I didn't notice that.